screwdriver.yaml

```diff
@@ -0,0 +1,15 @@
+version: 4
+shared:
+  steps:
+      - prevalidate_code: cd package
+      - postvalidate_code: cd $SD_SOURCE_DIR
+
+jobs:
+  validate_test:
+    template: python/validate_unittest
```

Comment:
@dwighthubbard we don't have these templates available in https://cd.screwdriver.cd/templates

Reply:
we should get those templates in before merging this.

Reply:
Ok, the templates are now on  https://cd.screwdriver.cd/templates